### ISSUE TITLE
parallel hang in itaps test

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#2362]
Eric reports that sometimes, itaps test hangs in parallel. but, scalable parallel works alright.

I ran valgrind on the tests and see a slew of memory errors.

The itaps libs were updated recently and perhaps memory errors are causing problems for parllel execution.

Long story short, some cases in itaps.py are skipped in parallel by adding logic to the .py file. This is because just adding tests to skip list does NOT prevent actual execution (only diffing logic) and so skip list cannot be used to avoid the hang.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Mark Miller
Original creation: 08/10/2010 11:06 pm
Original update: 08/18/2010 05:18 pm
Ticket number: 340
